server/channels/app/file.go

```diff
@@ -733,7 +735,8 @@ func (t *UploadFileTask) init(a *App) {
 // upload, returning a rejection error. In this case FileInfo would have
 // contained the last "good" FileInfo before the execution of that plugin.
 func (a *App) UploadFileX(c request.CTX, channelID, name string, input io.Reader,
-	opts ...func(*UploadFileTask)) (*model.FileInfo, *model.AppError) {
+	opts ...func(*UploadFileTask),
```

Comment:
Nit: It's easier to read if `opts` is part of the same line as the other arguments.

Reply:
Fixed